File: Controller/FunctionController.cs
using ComponentFactory.Krypton.Toolkit;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Isocline.Controller
{
    internal class FunctionController
    {
        private KryptonTextBox box_fxy;
        public FunctionController(KryptonTextBox box_fxy) 
        {
            this.box_fxy = box_fxy;
        } 
        public Func<double, double, double> FuncParser(String experssion)
        {
            var f = FunctionParser.CreateFXY(box_fxy.Text);
            return f;
        }
    }
}

File: Controller/FunctionController.vb
Imports ComponentFactory.Krypton.Toolkit
Imports System

Namespace Isocline.Controller
    Friend Class FunctionController
        Private box_fxy As KryptonTextBox
        Public Sub New(ByVal box_fxy As KryptonTextBox)
            Me.box_fxy = box_fxy
        End Sub
        Public Function FuncParser(ByVal experssion As String) As Func(Of Double, Double, Double)
            Dim f = FunctionParser.CreateFXY(box_fxy.Text)
            Return f
        End Function
    End Class
End Namespace
